src/aat/resources/features/F-102/F-102.feature

```diff
@@ -0,0 +1,49 @@
+@F-102
+Feature: Get jurisdictions available to the user
```

Comment:
Can we prefix feature with number, like we have done previously to be consistent so 

Feature: F-102: Get jurisdictions available to the user